ducktape/src/main/scala/io/github/arainko/ducktape/internal/macros/ProductTransformations.scala

```diff
@@ -132,22 +132,21 @@ private[ducktape] object ProductTransformations {
   private def fieldTransformations[Source: Type](
     sourceValue: Expr[Source],
     fieldsToTransformInto: List[Field]
-  )(using Quotes, Fields.Source) = {
+  )(using Quotes, Fields.Source): List[quotes.reflect.NamedArg] = {
```

Comment:
just my style thingy, I'd rather avoid type ascriptions in `private` methods (especially when returning a value of a path dependent type) - it just really noisy